### PR TITLE
Implementation - test connection

### PR DESCRIPTION
**Pull request by michaljurecko**
Jira: https://keboola.atlassian.net/browse/COM-545

### CODE REVIEW

**Review by ondrajodas — COMMENTED** [review]
jeden dotaz

File: tests/functional/test-connection-ok/source/data/config.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "action": "testConnection",
+  "parameters": {
+    "db": {
+      "#connectionString": "%env(string:CONNECTION_STRING)%"
```
Comment (ondrajodas) [inline tests/functional/test-connection-ok/source/data/config.json:5]: tam se bude zadávat celý connectionString?

Comment (michaljurecko) [reply tests/functional/test-connection-ok/source/data/config.json:5]: Ano, je to z 2 dovodov:
1. Priamo Azure v UI aj vsetkych tutorialoch uvadza `connection string`, ... teda nechceme po uzivatelovi aby to rucne parsoval (aj ked v screenshote je to aj oddelene, ale na inych miestach UI je len connection string).
2. Extractor funguje aj pre `ComsosDB Table API` aj pre `Storage Table`, kedze maju rovnaky interface,  ... no format connection string sa moze lisit.

screenshot-portal azure com-2020 11 30-{time)

**Review by ondrajodas — APPROVED** [review]
ok 👍

**Pull request merged**